jgrapht-core/src/main/java/org/jgrapht/generate/NamedGraphGenerator.java

```diff
@@ -0,0 +1,1033 @@
+            addEdge(targetGraph, edge[0], edge[1]);
+    }
+
+    //-------------Clebsch Graph-----------//
+    /**
+     * @see #generateClebschGraph
+     * @return Clebsch Graph
+     */
+    public static Graph<Integer, DefaultEdge> clebschGraph(){
+        Graph<Integer, DefaultEdge> g=new SimpleGraph<>(DefaultEdge.class);
+        new NamedGraphGenerator<Integer, DefaultEdge>(new IntegerVertexFactory()).generateClebschGraph(g);
+        return g;
+    }
+    /**
+     * Generates a <a href="http://mathworld.wolfram.com/ClebschGraph.html">Clebsch Graph</a>.
+     * The Clebsch graph, also known as the Greenwood-Gleason graph (Read and Wilson, 1998, p. 284), is a strongly
+     * regular quintic graph on 16 vertices and 40 edges.
+     * @param targetGraph receives the generated edges and vertices; if this is non-empty on entry, the
+     *        result will be a disconnected graph since generated elements will not be connected to
+     *        existing elements
+     */
+    public void generateClebschGraph(Graph<V,E> targetGraph){
+        vertexMap.clear();
+        int x = 0;
+        for(int i=0; i<8; i++) {
+            addEdge(targetGraph, x % 16, (x + 1) % 16);
+            addEdge(targetGraph, x % 16, (x + 6) % 16);
+            addEdge(targetGraph, x % 16, (x + 8) % 16);
+            x++;
+            addEdge(targetGraph, x % 16, (x + 3) % 16);
+            addEdge(targetGraph, x % 16, (x + 2) % 16);
+            addEdge(targetGraph, x % 16, (x + 8) % 16);
+            x++;
+        }
+    }
+
+    //-------------Grötzsch Graph-----------//
+    /**
+     * @see #generateGrötzschGraph
+     * @return Grötzsch Graph
+     */
+    public static Graph<Integer, DefaultEdge> grötzschGraph(){
+        Graph<Integer, DefaultEdge> g=new SimpleGraph<>(DefaultEdge.class);
+        new NamedGraphGenerator<Integer, DefaultEdge>(new IntegerVertexFactory()).generateGrötzschGraph(g);
+        return g;
+    }
+    /**
+     * Generates a <a href="http://mathworld.wolfram.com/GroetzschGraph.html">Grötzsch Graph</a>.
+     * The Grötzsch graph is smallest triangle-free graph with chromatic number four.
+     * @param targetGraph receives the generated edges and vertices; if this is non-empty on entry, the
+     *        result will be a disconnected graph since generated elements will not be connected to
+     *        existing elements
+     */
+    public void generateGrötzschGraph(Graph<V,E> targetGraph){
+        vertexMap.clear();
+        for(int i=1; i<6; i++)
+            addEdge(targetGraph, 0, i);
+        addEdge(targetGraph, 10, 6);
+        for(int i=6; i<10; i++){
+            addEdge(targetGraph, i, i+1);
+            addEdge(targetGraph, i, i-4);
+        }
+        addEdge(targetGraph, 10, 1);
+        for(int i=7; i<11; i++)
+            addEdge(targetGraph, i, i-6);
+        addEdge(targetGraph, 6, 5);
+    }
+    ///////////////////////////////////////////////
+
+    //-------------Hall-Janko Graph-----------//
```

Comment:
We could indeed move the NamedGraphGenerator to jgrapht-io. That way we indeed have access to the importers. However, this solution has some major disadvantages. One of the reasons to have access to well-known, named graphs, is to simplify (and improve) our unit tests in jgrapht-core. I don't think it's possible to access the named graphs from within the test classes in jgrapht-core when the NamedGraphGenerator is located in jgrapht-io.

Here's an example: PR #416 proposes a new planarity algorithm. Several of the named graphs are known to be non-trivial planar graphs. One could now simply write:
```
@Test
public void testFranklinGraph(){
    Graph<Integer, DefaultEdge> g=NamedGraphGenerator.franklinGraph();
    assertTrue(GraphTests.isPlanar(g));
}
```
Obviously, there are many more examples. Having access to the Named Graphs in jgrapht-core test suite is in my opinion one of the biggest benefits.